reaktive/src/jsMain/kotlin/com/badoo/reaktive/utils/UncaughtErrorHandler.kt

```diff
@@ -0,0 +1,4 @@
+package com.badoo.reaktive.utils
+
+actual var reaktiveUncaughtErrorHandler: (Throwable) -> Unit =
+    { e -> console.error(e) }
```

Comment:
Method reference

Reply:
`console.error(vararg o: Any?): Unit` cannot be used as a method reference for `(Throwable) -> Unit`